### ISSUE TITLE
Datatable filtered export bug

### ISSUE THREAD

**Issue by pantonis** [primefaces/primeng#2525]
**I'm submitting a ...**  (check one with "x")
```
[ X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Please fork the plunkr below and create a case demonstrating your bug report. Issues without a plunkr have much less possibility to be reviewed.

http://plnkr.co/edit/77OG7zX3LToGoidKaKOS

**Current behavior**
When trying to export a filtered datatable all data are exported 

**Expected behavior**
It should only export filtered data

**Minimal reproduction of the problem with instructions**
Load a datatable with filtering and export functionality. Then type in a filter and then try to export

**Please tell us about your environment:**
Windows 10, Vs Code, 

* **Angular version:** 4.0.0

* **PrimeNG version:** 2.0.6

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
In all browsers
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5] TypeScript 2.2.2

* **Node (for AoT issues):** `node --version` =   6.9.1

**Comment by pantonis** [comment]
I installed 4.0.0 Still getting the same problem. I have checked the source code the changes you commited are not there.

**Comment by pantonis** [comment]
I can confirm that this was fixed in 4.0.1

Thank you
